src/components/Banner.js

```diff
@@ -32,19 +61,35 @@ const Banner = props => (
                 styles.borderRadiusNormal,
                 isHovered ? styles.activeComponentBG : styles.hoveredComponentBG,
                 styles.breakAll,
+                ...props.containerStyles,
             ]}
             >
-                <View style={[styles.mr3]}>
-                    <Icon
-                        src={Expensicons.Exclamation}
-                        fill={StyleUtils.getIconFillColor(getButtonState(isHovered))}
-                    />
+                <View style={[styles.flexRow, styles.flexGrow1, styles.mw100, styles.alignItemsCenter]}>
+                    {props.shouldShowIcon && (
+                        <View style={[styles.mr3]}>
+                            <Icon
+                                src={Expensicons.Exclamation}
+                                fill={StyleUtils.getIconFillColor(getButtonState(isHovered))}
+                            />
+                        </View>
+                    )}
+                    {
+                        props.shouldRenderHTML
+                            ? <RenderHTML html={props.text} />
+                            : <Text style={[...props.textStyles]} onPress={props.onPress}>{props.text}</Text>
```

Comment:
Hey, just noting that we should have used flexWrap  to prevent text overflow on native.

https://github.com/Expensify/App/issues/42373#issuecomment-2120038486